circuits/cpp/barretenberg/cpp/src/barretenberg/ecc/curves/grumpkin/grumpkin.cpp

```diff
@@ -4,7 +4,9 @@ namespace grumpkin {
 namespace {
 
 constexpr size_t max_num_generators = 1 << 10;
+// NOLINTNEXTLINE TODO(@zac-williamson) get rid of need for these static variables in Pedersen refactor!
```

Comment:
Yes please. Can you link to the pedersen refactor issue/PR?

Reply:
done